Components/alert/alert.css

```diff
@@ -0,0 +1,47 @@
+.alert{
+    display: flex;
+    width: 40%;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 1px 1px 2px 2px rgba(0,0,0,0.25);
+    border-radius: 5px;
+    min-width: 40rem;
+    padding: 0.5em 1rem;
+    margin: 1.6rem 1.6rem;
+}
+
+.alerts-bold-weight{
+    font-size: 1.6rem;
+    margin-left: 5px;
+}
+
+.alerts-para{
+    font-size: 1.6rem;
+    font-weight: 300;
+    margin-left: 5px;
+}
+.alert-cross{
+    background-color: white;
+    margin-left: auto;
+}
+.alert i{
+    font-size: 2.4rem;
+}    
+
+.alert-success i{
+    color: #60C060;
+}
+
+
+.alert-info i{
+    color: #4D90Fd;;
+}
+
+
+.alert-notification i{
+    color: #FC9700;
+}
+
+.alert-error i{
+    color:red;

```

Comment:
user color variables.

For Dark theme you might want to use darker shade of red. Changing at 10 different places in project is not good idea.

Reply:
I will update it